### DISCUSSION TITLE
Using BitPb without server project

### DISCUSSION THREAD

**Discussion by taha-ghadirian** [bitfoundation/bitplatform#6276] [Q&A] [answered]
Hi, 

It's been a while that I haven't used BitPlatform, But now I'm switched back and noticed some amazing updates! 
However, I'm now struggling with this new templates, because in older templates, you could run the project without `Server` side project, you could easily publish `Client.Web` as WebAssembly file and use it in legacy static content hosting (Such as `Github Pages` using either manual deployment by coping the publish folder that contained `index.html` file or using Ci/Cd).

But in new templates, whole project depends on Server project, that means there in no direct way to use Client side only as a single project. 
I hope the community can help me to solve this issue :)
Thanks.

**Comment by taha-ghadirian** [comment]
Closed as duplicate #5217 
However, I think this would be great if the feature back to templates.

**Comment by yasmoradi** [comment, marked as answer]
Welcome back!
The sdk of `Web` project is `Microsoft.NET.Sdk.BlazorWebAssembly` and you can always enable debugging by simply adding `index.html` and `launchsettings.json` to it.
Adding index.html also makes that static folder / without server deployment possible.
